### DISCUSSION TITLE
Enable `requireConfirm` for `onSave` and disable it for `onClose`

### DISCUSSION THREAD

**Discussion by davidgarra** [mia-platform/community#357] [Q&A] [answered]
### Which section?

Backoffice

### Which is the version?

back-kit@1.4.7, microlc-backend@0.8.1, microlc-frontend@0.9.0

### Description

Hello everyone, I'm having trouble configuring a `bk-dynamic-form-modal` or a 'bk-dynamic-form-drawer' I can't find a way to enable `requireConfirm` only for `onSave` and disable it for `onClose`. I tried setting the properties as follows:
```json
   {
      "type": "element",
      "tag": "bk-dynamic-form-modal",
      "properties": {
        // ... other props
        "requireConfirm": {
          "onSave": true,
          "onClose": false
        },
      }
    },
```
Despite this configuration, the form persists in requesting confirmation upon closure.

Is this correct? How can I disable it just for the `onClose`?

**Comment by clabene** [comment, marked as answer]
Hi @davidgarra

There is a mistake in our documentation - we are fixing it asap, apologies for the inconvenience.

The configuration you are looking for is:
```
        "requireConfirm": {
          "save": true,
          "cancel": false
        }
```

Please also note that, as of now, a form that is holding unsaved changes always asks for confirmation upon closing.
We are planning to extend this behavior in future releases to allow a more fine grained configuration.

**Reply by davidgarra** [reply]
Thanks for the info, the suggested configuration works perfectly. Thank you
